service/src/IOUring.cpp

```diff
@@ -34,7 +36,8 @@ namespace geopm
     std::unique_ptr<IOUring> IOUring::make_unique(unsigned entries)
     {
 #ifdef GEOPM_HAS_IO_URING
-        if (IOUringImp::is_supported()) {
+        if (IOUringImp::is_supported() &&
+            std::getenv("GEOPM_DISABLE_IO_URING") == nullptr) {
```

Comment:
Same suggestion as in IOGroup.cpp here.

Reply:
Same argument applies that this is preferred to geopm::get_env().